pkg/retriever/retriever_test.go

```diff
@@ -31,96 +31,148 @@ func TestFilterersEventList(t *testing.T) {
 	}
 }
 
-// TestEventCollection tests that events are being collected,
-// TODO: make better test here w simulated backend
-// XXX(PN): Infura has been having problems, so commenting this out so builds work.
```

Comment:
Is this test invalid now? This was commented out due to Infura issues, but probably can be re-enabled.

Reply:
I just changed the name of the test to "TestRetrieveMethod" since it's not a test related to the event collector